packages/plugins/i18n/admin/src/translations/de.json

```diff
@@ -0,0 +1,64 @@
+{
+  "CMEditViewCopyLocale.copy-failure": "Beim Kopieren der Übersetzungen ist ein Fehler aufgetreten",
+  "CMEditViewCopyLocale.copy-success": "Übersetzungen kopiert",
+  "CMEditViewCopyLocale.copy-text": "Von anderer Übersetzung übernehmen",
+  "CMEditViewCopyLocale.submit-text": "Ja, ausfüllen",
+  "CMListView.popover.display-locales.label": "Übersetzungen anzeigen",
+  "CheckboxConfirmation.Modal.body": "Möchtest du die Lokalisierung deaktivieren?",
+  "CheckboxConfirmation.Modal.button-confirm": "Ja, deaktivieren",
+  "CheckboxConfirmation.Modal.content": "Wenn Du die Lokalisierung deaktivierst, werden alle Ihre Inhalte mit Ausnahme des Inhalts, der Ihrem Standardgebietsschema zugeordnet ist (sofern vorhanden), gelöscht.",
+  "Field.localized": "Dieser Wert ist einzigartig für die ausgewählte Übersetzung",
+  "Field.not-localized": "Dieser Wert ist für alle Übersetzungen gleich",
+  "Settings.list.actions.add": "Neue Übersetzung hinzufügen",
+  "Settings.list.actions.delete": "Übersetzung löschen",
+  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen.",
```

Comment:
```suggestion
  "Settings.list.actions.deleteAdditionalInfos": "Dies wird die aktive Übersetzung löschen <em>(aus der Internationalisierung)</em>",
```

Reply:
I know I omitted the last part here, but to be honest, I don't get it. Of course I am not deleting French from it's existence, but I am deleting it from the i18n of the current strapi instance. Is there a reason why this is specifically stated?

Reply:
TBH I am not too sure why its stated. But maybe it makes sense to keep it though - probably it is there for some reason.

Reply:
I think we can leave this one out tbh.